packages/react-components/src/components/SendBox.tsx

```diff
@@ -156,6 +158,12 @@ export interface SendBoxProps {
    * @beta
    */
   activeFileUploads?: ActiveFileUpload[];
+  /**
+   * Optional callback to remove the file upload before sending by clicking on
+   * cancel icon.
+   * @beta
+   */
+  cancelFileUpload?: (fileId: string) => void;
```

Comment:
It should be named `onCancelFileUpload` since that's the convention we are using throughout our components.

Reply:
We might need to pass in `userId` as a param too.

Reply:
`userId` shouldn't be needed since this is always for the local participant, and we always assume that Contoso has the local participant's `userId`.

Reply:
Right. Makes sense.

Reply:
Done